sorc/chgres_cube.fd/surface.F90

```diff
@@ -2234,6 +2609,518 @@ subroutine calc_liq_soil_moisture
 
```

Comment:
FYI.  calc_liq_soil_moisture was written to support the Noah-LSM.  It is likely not relevant for RUC-LSM.  I know nothing about land models other than Noah.   So expanding chgres_cube to work with input data sets with land states from other land models will need to be carefully done.

Reply:
HRRR data is indeed RUC LSM data, but the soil types are the same classification system so I assumed this would still be relevant. However, I do believe the RUC LSM states (particularly how RUC treats soil type under land ice) are the cause of the soil moisture mismatches that you mentioned in a prior question.